en/resources/tutorials.md

```diff
@@ -65,6 +65,12 @@ This tutorial shows how to use popular templating engines like Mustache and Sten
 
 Kitura supports FastCGI 1.0 and has been tested with both Nginx and Apache.
 
+---
+
+## Dev Ops
+
+### [Generating the Router verb code](/{{ page.lang }}/resources/tutorials/router_verb_gen.html)
```

Comment:
This is not relevant to Kitura users - only Kitura developers would ever need to do this. So I don't think it belongs on `kitura.io`.